src/Uno.UI/UI/Xaml/Internal/InputManager.Pointers.Managed.cs

```diff
@@ -0,0 +1,435 @@
+﻿#if UNO_HAS_MANAGED_POINTERS
+#nullable enable
```

Comment:
@dr1rrb

Reply:
I assume that all of this is coming from the `UIelement.Pointers.managed.cs` ? You didn't have to change the code itself @MartinZikmund  ?

Reply:
Yes, I just had to move it from UIElement, but it is verbatim copy